frontend/src/app/components/ExperimentPage.js

```diff
@@ -147,6 +147,10 @@ export default class ExperimentPage extends React.Component {
             introduction, measurements, privacy_notice_url, changelog_url,
             thumbnail, survey_url, contributors, details, min_release } = experiment;
 
+    // Set the timestamp for when this experiment was last seen (for
+    // ExperimentRowCard updated/launched banner logic)
+    setExperimentLastSeen(experiment);
```

Comment:
We lost this from ye olde Ampersand site, [over here](https://github.com/mozilla/testpilot/blob/2016-09-12/frontend/src/app/views/experiment-page.js#L272). Just realized it wasn't working as I wrote the test.